import 使用重命名，就挂了

import { SomeCom  as Com } from 'antd'

转换后

var Com = require('antd/Com')

而不是
var Com = require('antd/some-com')

`plugins: [
    [
      'component',
      {
        libraryName: 'mint-ui',
        style: true
      },
      'mint-ui'
    ],
    [
      'component',
      {
        libraryName: 'element-ui',
        styleLibraryName: 'theme-chalk'
      },
      'element-ui'
    ]
  ]`



`import { Button as mtButton } from 'mint-ui'`
`import { Button as elButton } from 'element-ui'`

`Error: Cannot find module 'element-ui/lib/el-button'`
`Error: Cannot find module 'mint-ui/lib/mt-button'`

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "debug": "umi-tools test",
     "lint": "eslint --ext .js src",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "prepack":  "npm run build"
+    "prepack": "npm run build"
   },
   "pre-commit": [
     "lint"

--- a/src/Plugin.js
+++ b/src/Plugin.js
@@ -111,7 +111,7 @@ export default class Plugin {
     if (pluginState.specified[node[prop].name] &&
       path.scope.hasBinding(node[prop].name) &&
       path.scope.getBinding(node[prop].name).path.type === 'ImportSpecifier') {
-      node[prop] = this.importMethod(node[prop].name, file, pluginState);  // eslint-disable-line
+      node[prop] = this.importMethod(pluginState.specified[node[prop].name], file, pluginState);  // eslint-disable-line
     }
   }
 

--- a/test/fixtures/variable-declarator-renamed-import/actual.js
+++ b/test/fixtures/variable-declarator-renamed-import/actual.js
@@ -1,0 +1,5 @@
+import { Button as AntButton } from 'antd';
+
+const Button = 'Another Button';
+
+const a = AntButton;

--- a/test/fixtures/variable-declarator-renamed-import/expected.js
+++ b/test/fixtures/variable-declarator-renamed-import/expected.js
@@ -1,0 +1,8 @@
+"use strict";
+
+var _button = _interopRequireDefault(require("antd/lib/button"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+var Button = 'Another Button';
+var a = _button.default;

